books/clients/livelibru.py

```diff
@@ -0,0 +1,89 @@
+import requests
+import urllib
+import json
+import math
+
+REQUIRED_HEADERS = {
+    "Host": "www.livelib.ru",
+    "Accept-Language": "en-US;q=1.0, ru-US;q=0.9, be-US;q=0.8",
+    "Accept": "*/*",
+    "Connection": "keep-alive",
+    "Accept-Encoding": "gzip;q=1.0, compress;q=0.5",
+    "User-Agent": "LiveLib/3.24.5 (ru.livelib.LiveLib; build:3.24.5; iOS 16.5.1) Alamofire/3.24.5"
```

Comment:
Can you add a comment here explaining where you got this useragent from? In future we might need to update it so providing context how to get it would be useful.